azure-pipelines.yml

```diff
@@ -176,98 +173,162 @@ steps:
-  #     in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
-  #     ne(variables['Build.Reason'], 'PullRequest'),
-  #     ne(variables['Build.Reason'], 'Manual')
-  #     )
-  #   inputs:
-  #     SourcePath: '$(Build.ArtifactStagingDirectory)\$(prepNextRelease.NextVersion)'
-  #     ConnectionType: 'EnvironmentVariables'
-  #     Environment: 'PublicCloud'
-  #     ContainerName: 'teams-js'
-  #     AdditionalArgumentsForBlobCopy: '--overwrite=false --log-level=INFO --recursive'
-  #   env:
-  #     M365CDN_APP_ID: $(M365CDNAppIdProd)
-  #     M365CDN_APP_KEY: $(CDNAppKeyProd)
-  #     M365CDN_TENANT_ID: $(M365CDNTenantId)
-  #   displayName: 'Push TeamsJS to M365 1CDN(Prod) for (Main)'
-  # IMPORTANT
-  # If you need to change $(secret.private-preview-ado-token), you need to base64 encode the string given to you when creating the Personal Access Token, even if it looks like its already base64 encoded.
-  # - script: 'node tools/cli/deployPrivatePreview.js'
-  #   condition: and(
-  #       startsWith(variables['Build.SourceBranch'], 'refs/tags/v'),
-  #       in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
-  #       ne(variables['Build.Reason'], 'PullRequest'),
-  #       ne(variables['Build.Reason'], 'Manual')
-  #     )
-  #   env:
-  #     PRIVATE_PREVIEW_TOKEN: $(secret.private-preview-ado-token)
-  #   displayName: 'yarn publish-sdk (private preview)'
\ No newline at end of file
+     SourcePath: '$(Build.ArtifactStagingDirectory)\CDNFeed\$(package.Version)'
+     ConnectionType: 'EnvironmentVariables'
+     Environment: 'PublicCloudSDF'
+     ContainerName: 'teams-js'
+     AdditionalArgumentsForBlobCopy: '--overwrite=false --log-level=INFO --recursive'
+    env:
+      M365CDN_APP_ID: $(M365-APP-CLIENT-ID)
+      M365CDN_APP_KEY: $(M365-APP-SECRET)
+      M365CDN_TENANT_ID: $(M365-APP-TENNANT-ID)
+    displayName: 'Push TeamsJS to M365 1CDN(SDF) for (2.0-preview CI/CD)'
+    
+  # This task will push content to CDN-Prod for every prod release.
+  # Sample CDN URL: https://res.cdn.office.net/teams-js/2.0.0-beta.0/js/MicrosoftTeams.min.js
+  - task: M365CdnAssetsUpload@1
+    condition: and(
+      startsWith(variables['Build.SourceBranch'], 'refs/tags/v'),
+      in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
+      ne(variables['Build.Reason'], 'PullRequest'),
+      ne(variables['Build.Reason'], 'Manual')
+      )
+    inputs:
+      SourcePath: '$(Build.ArtifactStagingDirectory)\CDNFeed\$(package.Version)'
+      ConnectionType: 'EnvironmentVariables'
+      Environment: 'PublicCloud'
+      ContainerName: 'teams-js'
+      AdditionalArgumentsForBlobCopy: '--overwrite=false --log-level=INFO --recursive'
+    env:
+      M365CDN_APP_ID: $(M365-CDN-CLIENT-ID-PROD)
+      M365CDN_APP_KEY: $(M365-CDN-CLIENT-SECRET-PROD)
+      M365CDN_TENANT_ID: $(M365-APP-TENNANT-ID)
+    displayName: 'Push TeamsJS to M365 1CDN(Prod) for (Main)'
```

Comment:
Can we change this to 'Push TeamsJS to M365 1CDN (Prod) for (2.0-preview release)'?